llama-index-core/llama_index/core/ingestion/api_utils.py

```diff
@@ -1,151 +1,37 @@
 import os
-from typing import List, Optional
+from typing import Optional
 
-from llama_index_client import (
-    ConfigurableDataSourceNames,
-    ConfigurableTransformationNames,
-    ConfiguredTransformationItem,
-    DataSourceCreate,
-    PipelineCreate,
-    PipelineType,
-    ProjectCreate,
-)
-from llama_index_client.client import AsyncPlatformApi, PlatformApi
+from llama_cloud.client import AsyncLlamaCloud, LlamaCloud
 
 from llama_index.core.constants import (
     DEFAULT_APP_URL,
     DEFAULT_BASE_URL,
-    DEFAULT_PROJECT_NAME,
-)
-from llama_index.core.ingestion.data_sources import (
-    ConfiguredDataSource,
 )
-from llama_index.core.ingestion.transformations import (
-    ConfiguredTransformation,
-)
-from llama_index.core.node_parser import SentenceSplitter
-from llama_index.core.readers.base import ReaderConfig
-from llama_index.core.schema import BaseNode, TransformComponent
-
-
-def default_transformations() -> List[TransformComponent]:
-    """Default transformations."""
-    from llama_index.embeddings.openai import OpenAIEmbedding  # pants: no-infer-dep
-
-    return [
-        SentenceSplitter(),
-        OpenAIEmbedding(),
-    ]
 
 
 def get_client(
     api_key: Optional[str] = None,
     base_url: Optional[str] = None,
     app_url: Optional[str] = None,
```

Comment:
what's `app_url` for? seems like its not being used?

Reply:
previously, this was used to print link to go to the webapp.